README.md

```diff
@@ -13,6 +13,35 @@ If you have questions, visit us on irc.freenode.net in #opensuse-factory or
 ask on our mailing list opensuse-factory@opensuse.org
 
 
+### Coding style
+
+The project follows the rules of the parent project
+[os-autoinst](https://github.com/os-autoinst/os-autoinst#how-to-contribute).
+and additionally the following rules:
+
+* Take
+  [example boot.pm](https://github.com/os-autoinst/os-autoinst-distri-example/blob/master/tests/boot.pm)
```

Comment:
not good idea , need to point to something static here . you can't guarantee that after some time someone will not introduce some anti-pattern into this file. Copy-paste file to README ?